custom_components/smartthinq_sensors/switch.py

```diff
@@ -197,7 +217,7 @@ def available(self) -> bool:
         is_avail = True
         if self.entity_description.available_fn is not None:
             is_avail = self.entity_description.available_fn(self._wrap_device)
-        return self._api.available and self._wrap_device.is_power_on and is_avail
+        return self._api.available and is_avail
```

Comment:
This will break functionality for other devices, should not be changed

Reply:
self._wrap_device.is_power_on will be False when the air purifier is not running, this makes the switch become unavailable and cannot turn on the device by it

Reply:
Ok, understand the problem, you are right!
I changed definition for others switch to manage this situation.
Please rebase your PR, this change is already committed in the main branch.